SummerProject/config/settings.py

```diff
@@ -80,8 +82,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'summer_pr_db',
-        'USER': POSTGRES_DB_USERNAME,
-        'PASSWORD': POSTGRES_DB_PASSWORD,
+        'USER': 'milaviex',
```

Comment:
Please change this back to `POSTGRES_DB_USERNAME` and the line below to `POSTGRES_DB_PASSWORD`